Cargo.toml

```diff
@@ -16,9 +16,9 @@ Cargo subcommand to easily use LLVM source-based code coverage (-C instrument-co
 targets = ["x86_64-unknown-linux-gnu"]
 
 [package.metadata.binstall]
-pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.{ archive-format }"
+pkg-url = "{ repo }/releases/download/v{ version }/{ name }-{ target }.tar.gz"
 bin-dir = "{ bin }{ binary-ext }"
-pkg-fmt = "tar.gz"
+pkg-fmt = "tgz"
```

Comment:
(It seems that pkg-fmt means compression format, but it is unfortunate that commonly used extensions such as tar.gz are not supported as aliases.)